src/test/kotlin/no/nav/faktureringskomponenten/service/beregning/BeløpBeregnerTest.kt

```diff
@@ -151,4 +152,59 @@ class BeløpBeregnerTest {
             BeløpBeregner.beløpForPeriode(BigDecimal("1002.25"), fom, tom)
         }
     }
+
+    @Test
+    fun `Totalbeløp, En periode for hele måneder`() {
+        val fom = LocalDate.of(2023, 1, 1)
+        val tom = LocalDate.of(2023, 5, 31)
+        val fakturaseriePeriode = FakturaseriePeriode(BigDecimal.valueOf(700), fom, tom, "dummy")
+        val fakturaseriePerioder = listOf(fakturaseriePeriode)
+
+        val result = BeløpBeregner.totalBeløpForAllePerioder(fakturaseriePerioder)
+
+        val forventetBeløp = BigDecimal("3500.00")
+        result.shouldBe(forventetBeløp)
+    }
+
+    @Test
+    fun `Totalbeløp, Ulike perioder med ulik trygdeavgift for hele måneder`() {
+        val fom = LocalDate.of(2023, 1, 1)
+        val tom = LocalDate.of(2023, 12, 31)
+        val fakturaseriePeriode = FakturaseriePeriode(BigDecimal.valueOf(500), fom, tom, "dummy")
+        val fom2 = LocalDate.of(2023, 6, 1)
+        val tom2 = LocalDate.of(2023, 12, 31)
+        val fakturaseriePeriode2 = FakturaseriePeriode(BigDecimal.valueOf(1000), fom2, tom2, "dummy")
+        val fakturaseriePerioder = listOf(fakturaseriePeriode, fakturaseriePeriode2)
+
+        val result = BeløpBeregner.totalBeløpForAllePerioder(fakturaseriePerioder)
+
+        val forventetBeløp = BigDecimal("13000.00")
+        result.shouldBe(forventetBeløp)
+    }
+
```

Comment:
Bra med testing, men her er det litt for mange tester ift. kompleksitet.
Tester er også kode som må vedlikeholdes, så jeg foreslår å fjerne noen av disse

Reply:
Kombinerte 3 tester til 1